index.html

```diff
@@ -107,17 +107,15 @@ <h3 class="slogan-our-mission">Just the right food</h3>
 
     <section class="ingredients-section">
       <div class="dark-square">
-        <figure>
-          <figcaption>
-            <h3 class="slogan-ingredients-section">Fine ingredients</h3>
-          </figcaption>
-          <img src="gallery/decoration/decoration-3.png" class="decoration-ingredients" alt="decorative element">
-          <blockquote class="description-ingredients-section">If you’ve been to one of our restaurants, you’ve seen –
-            and tasted – what keeps our customers coming back for more. Perfect materials and freshly baked food,
-            delicious
-            Lambda cakes, muffins, and gourmet coffees make us hard to resist! Stop in today and check us out!
-          </blockquote>
-        </figure>
+        <figcaption>
+          <h3 class="slogan-ingredients-section">Fine ingredients</h3>
+        </figcaption>
+        <img src="gallery/decoration/decoration-3.png" class="decoration-ingredients" alt="decorative element">
+        <blockquote class="description-ingredients-section">If you’ve been to one of our restaurants, you’ve seen –
+          and tasted – what keeps our customers coming back for more. Perfect materials and freshly baked food,
+          delicious
+          Lambda cakes, muffins, and gourmet coffees make us hard to resist! Stop in today and check us out!
+        </blockquote>
```

Comment:
These should be inside the parent <figure> element along with the images.

Reply:
understood what it was about.  fix it soon